Multisig/Data/Services/Safe Client Gateway Service/SCGModels.swift

```diff
@@ -538,6 +538,42 @@ extension SCGModels {
         let blockExplorerUrl: URL
         let nativeCurrency: Currency
         let theme: Theme
+
+        enum CodingKeys: String, CodingKey {
+            case chainId
+            case chainName
+            case rpcUrl
+            case blockExplorerUrl
+            case nativeCurrency
+            case theme
+        }
+
+        init(chainId: Int,
+             chainName: String,
+             rpcUrl: URL,
+             blockExplorerUrl: URL,
+             nativeCurrency: Currency,
+             theme: Theme) {
+            self.chainId = chainId
+            self.chainName = chainName
+            self.rpcUrl = rpcUrl
+            self.blockExplorerUrl = blockExplorerUrl
+            self.nativeCurrency = nativeCurrency
+            self.theme = theme
+        }
+
+        init(from decoder: Decoder) throws {
+            let values = try decoder.container(keyedBy: CodingKeys.self)
+            var string = try! values.decode(String.self, forKey: .chainId)
```

Comment:
I think if you'd define the value type as UInt256String, it will accept the stringified integer that can be used in other places, so we don't need to implement decodable here